cmd/srcd/cmd/gitbase_back_comp_test.go

```diff
@@ -0,0 +1,312 @@
+	expected = `+-------+------------+----------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+---------+------------+
+| TABLE | NON UNIQUE | KEY NAME | SEQ IN INDEX | COLUMN NAME | COLLATION | CARDINALITY | SUB PART | PACKED | NULL | INDEX TYPE | COMMENT | INDEX COMMENT | VISIBLE | EXPRESSION |
++-------+------------+----------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+---------+------------+
++-------+------------+----------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+---------+------------+
+`
+	require.NoError(cmdtest.AreSQLOutputEqual(buf.String(), expected))
+
+	// [previous version] srcd sql "CREATE INDEXES"
+	_, err = s.runSQL(s.PrevCmd, "CREATE INDEX repo_idx ON repositories USING pilosa (repository_id)")
+	require.NoError(err)
+	// wait a bit for index to be ready
+	time.Sleep(1 * time.Second)
+
+	// [previous version] srcd sql "SHOW INDEXES"
+	buf, err = s.runSQL(s.PrevCmd, "SHOW INDEX FROM repositories")
+	require.NoError(err)
+	expected = `+--------------+------------+----------+--------------+----------------------------+-----------+-------------+----------+--------+------+------------+---------+---------------+---------+------------+
+|    TABLE     | NON UNIQUE | KEY NAME | SEQ IN INDEX |        COLUMN NAME         | COLLATION | CARDINALITY | SUB PART | PACKED | NULL | INDEX TYPE | COMMENT | INDEX COMMENT | VISIBLE | EXPRESSION |
++--------------+------------+----------+--------------+----------------------------+-----------+-------------+----------+--------+------+------------+---------+---------------+---------+------------+
+| repositories |          1 | repo_idx |            1 | repositories.repository_id | NULL      |           0 |        0 | NULL   |      | pilosa     |         |               | YES     | NULL       |
++--------------+------------+----------+--------------+----------------------------+-----------+-------------+----------+--------+------+------------+---------+---------------+---------+------------+
+`
+	require.NoError(cmdtest.AreSQLOutputEqual(buf.String(), expected))
+
+	// [previous version] srcd sql "select * from repositories"
+	buf, err = s.runSQL(s.PrevCmd, "select * from repositories")
+	require.NoError(err)
+
+	expected = `+---------------+
+| REPOSITORY ID |
++---------------+
+| repo-0        |
+| repo-1        |
+| repo-2        |
++---------------+`
+	require.NoError(cmdtest.AreSQLOutputEqual(buf.String(), expected))
+
+	s.PrevCmd.RunStop(context.Background())
+
+	// [current version] srcd init s.testDir
+	_, err = s.runInit(s.CurrCmd, s.testDir)
+	require.NoError(err)
+
+	// [current version] srcd sql "select * from repositories"
+	buf, err = s.runSQL(s.CurrCmd, "select * from repositories")
+	require.NoError(err)
+
+	expected = `+---------------+
```

Comment:
if the purpose is to ensure that indexes could be used, why not checking it explicitly?
```sql
describe format=tree select repository_id from repositories where repository_id = 'repo-0'
```
should answer:
```
+-----------------------------------------------------+
|                      PLAN                           |
+-----------------------------------------------------+
| Exchange(parallelism=4)                             |
|  └─ Table(repositories)                             |
|      ├─ Column(repository_id, TEXT, nullable=false) |
|      ├─ Filters                                     |
|      │   └─ repositories.repository_id = "repo-0"   |
|      └─ Indexes                                     |
|          └─ repo_idx                                |
+-----------------------------------------------------+
```

Reply:
done